core/domain/user_id_migration.py

```diff
@@ -463,21 +466,13 @@ def _check_id_and_user_id_exist(model_id, user_id):
             return True
         return user_models.UserSettingsModel.get_by_id(user_id) is not None
 
-    @classmethod
```

Comment:
Keep this but have it raise NotImplementedError.

Reply:
Done.